src.ts/transaction/transaction.ts

```diff
@@ -535,13 +535,34 @@ export class Transaction implements TransactionLike<string> {
             return v.toString();
         };
     
-        // Adjusted function to specifically handle the conversion of 'denomination' fields in array items
-        const processArrayWithBigInt = (arr: UTXOEntry[] | UTXOTransactionOutput[]) => {
-            return arr.map(item => ({
-                address: item.address,
-                denomination: s(item.denomination) // Convert 'denomination' to string
-            }));
+        // Helper function to convert bigint or number to string for JSON output
+        const bigIntToString = (value: number | bigint): string => {
```

Comment:
Since this method is a single line of code and only used once, we should remove it and use value.toString() for the value of `Denominantion`